sequence_task_1/student/include/impl.h

```diff
@@ -8,33 +8,86 @@ class StaticPlaylist
 {
 public:
     /** @todo Member traits */
+    typedef Song_t value_type;
+    typedef Song_t& reference;
+    typedef const Song_t& const_reference;
+    typedef typename Container::iterator iterator;
+    typedef typename Container::const_iterator const_iterator;
+    typedef ptrdiff_t difference_type;
+    typedef size_t size_type;
 
     /** @todo Iterators */
```

Comment:
what about non-const iterators? (sorry, I should add test for this)

Reply:
The test RangeFor.Iterators complaints about the elements being non-const if i add regular non-const iterators. That's why I didn't add them. Even if I mark the begin()/end() methods as const.

Reply:
so, it happens even we have 2 versions of iterators - const and non const?
I'll check and fix the test than, thanks